libraries/AP_ADSB/AP_ADSB_uAvionix_UCP.cpp

```diff
@@ -130,6 +130,56 @@ void AP_ADSB_uAvionix_UCP::update()
         && (now_ms - run_state.last_packet_Transponder_Ownship_ms >= 10000)) {
         _frontend.out_state.tx_status.fault |= UAVIONIX_ADSB_OUT_STATUS_FAULT_STATUS_MESSAGE_UNAVAIL;
     }
+
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
```

Comment:
This is not the right place for simulations.

AFAICS this should look much like the way we simulate the SageTech MXS.

Reply:
@peterbarker  yeah.. I started doing that and it was just sooooo much. Total overkill. This is actually a larger version of one I've been running on my fork for years. If we don't want this I'm happy to close this and hope someone does it the other verbose method where we're actually checking uart baudrates and CRCs and all the low-level stuff.

Reply:
Yep, cool.  Thanks anyway!